dotnet/src/Connectors/Connectors.OpenAI/Core/ClientCore.cs

```diff
@@ -10,6 +10,9 @@
 using System.Threading.Tasks;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Abstractions;
+#pragma warning disable IDE0005 // Using directive is unnecessary
+using Microsoft.SemanticKernel.Connectors.AI;
+#pragma warning restore IDE0005 // Using directive is unnecessary
```

Comment:
If you are using the AI namespace, this should be ok right?

```suggestion
using Microsoft.SemanticKernel.Connectors.AI;
```

Reply:
Suggest something like `Shared` or `Common` or `FunctionCalling` instead of `AI`.

Reply:
I don't see any problem using AI. I don't see any problem using one of the suggested options either. I'm open to change as long as I see the benefits of doing so.